1.0/docs/devguide/events.md

```diff
@@ -102,6 +102,41 @@ So the attribute `on-core-signal-newData` sets up a listener for `core-signal-ne
 _not_ `core-signal-newData`. To avoid confusion, always use lowercase event names.
 {: .alert .alert-info } 
 
+## Custom Events {#custom-events}
+
+To fire a custom event from the host element use `this.fire()`. You can also pass in data to event handlers as an argument to `this.fire()`.
```

Comment:
For consistency with other docs, this should be "the `fire` method." -- no `this.`, no parens unless you're listing arguments. (The second reference can just be `fire`.)